src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/BookingFacade.kt

```diff
@@ -102,7 +102,7 @@ class BookingFacade(
     contacts.mapNotNull { contact ->
       when (contact.contactType) {
         ContactType.USER -> CourtEmailFactory.user(contact, prisoner, booking, prison, main, pre, post, locations, eventType)
-        ContactType.COURT -> CourtEmailFactory.court(contact, prisoner, booking, prison, main, pre, post, locations, eventType)
+        ContactType.COURT -> if (booking.createdByPrison || eventType == BookingAction.RELEASED || eventType == BookingAction.TRANSFERRED) CourtEmailFactory.court(contact, prisoner, booking, prison, main, pre, post, locations, eventType) else null
```

Comment:
I still think we have this wrong. I need to think about this a bit more ...

To me it feels like perhaps we should be asking for the booking contacts based on the operation / user type and not just asking for all contact types.

We could perhaps even introduce another contact type ContactType.EXTERNAL ?  This would still require changes to the contacts service.

Will approve for now but I'd like to see this changed/refactored.